heppy/Doc.py

```
@@ -48,6 +49,7 @@ class Doc:
         'neulevel':     'urn:ietf:params:xml:ns:neulevel',
         'neulevel10':   'urn:ietf:params:xml:ns:neulevel-1.0',
         'kv':           'urn:X-ar:params:xml:ns:kv-1.0',
+        'sync':         'http://www.verisign.com/epp/sync-1.0',
```

Ruff (0.14.8)

Dictionary key literal `'sync'` repeated

Remove repeated key literal `'sync'`

(F601)